scripts/modules/elastic_stack.py

```diff
@@ -691,6 +691,8 @@ def __init__(self, **options):
             self.environment["XPACK_MONITORING_ENABLED"] = "true"
             if self.at_least_version("6.3"):
                 self.environment["XPACK_XPACK_MAIN_TELEMETRY_ENABLED"] = "false"
+            if self.at_least_version("7.7"):
+                self.environment["XPACK_SECURITY_ENCRYPTIONKEY"] = "fhjskloppd678ehkdfdlliverpoolfcr"
```

Comment:
![](https://media.giphy.com/media/VfDqzZ3Efb67HCN2n6/giphy.gif)

Reply:
I'm not certain how this key was chosen.  We used the example from the following guide.  Would you suggest an alternative approach @mdelapenya ?

https://www.elastic.co/guide/en/siem/guide/current/detection-engine-overview.html#detections-permissions

Reply:
Oh no no, I liked it 'cause I'm a Liverpool FC supporter!! 😄